frontend/views/containers/proposals/MemberRequest.vue

```diff
@@ -0,0 +1,250 @@
+    :title='L("Member Requests")'
+  )
+    .c-all-actions
+      i18n.button.is-outlined.is-small(
+        tag='span'
+        @click='approveAll'
+      ) Approve All
+      i18n.button.is-outlined.is-small(
+        tag='span'
+        @click='rejectAll'
+      ) Reject All
+
+    ul.c-group-list
+      li.c-group-member(
+        v-for='{username, displayName, status, date} in requestsSorted'
+        :data-test='username'
+        :key='username'
+      )
+        profile-card(:username='username')
+          avatar-user(:username='username' size='sm')
+          .c-name.has-text-bold {{username}}
+          .c-date.has-text-1 {{ humanDate(date, { month: 'long', day: 'numeric', year: 'numeric' }) }}
+          .c-action-container(v-if='status === "requested"')
+            i18n.button.is-outlined.is-small(
+              tag='span'
+              @click='approve'
+            ) Approve
+            i18n.button.is-outlined.is-small(
+              tag='span'
+              @click='reject'
+            ) Reject
+          .c-action-container.c-undo-container(v-else)
+            .has-text-danger(v-if='status === "rejected"')
+              i.icon-times
+              i18n(
+                tag='span'
+              ) Request Rejected.
+
+            .has-text-success(v-else)
+              i.icon-check
+              i18n(
+                tag='span'
+              ) Request Approved.
+
+            i18n.is-link-inherit.has-text-1(
+              tag='span'
+              @click='undo'
+            ) Undo
+
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import SvgConversation from '@svgs/conversation.svg'
+import CalloutCard from '@components/CalloutCard.vue'
+import ProposalBox from '@containers/proposals/ProposalBox.vue'
+import PageSection from '@components/PageSection.vue'
+import Avatar from '@components/Avatar.vue'
+import AvatarUser from '@components/AvatarUser.vue'
+import ProfileCard from '@components/ProfileCard.vue'
+import { humanDate } from '@utils/time.js'
+
+export default {
+  name: 'ProposalsWidget',
+  components: {
+    ProposalBox,
+    CalloutCard,
+    SvgConversation,
+    PageSection,
+    Avatar,
+    AvatarUser,
+    ProfileCard
+  },
+  data () {
+    return {
+      SvgConversation,
+      requestsSorted: [
+        {
+          username: 'Pierre',
+          date: new Date().toISOString(),
+          displayName: 'Pierre',
+          status: 'requested'
+        },
+        {
+          username: 'Pierre',
+          date: new Date().toISOString(),
+          displayName: 'Pierre',
+          status: 'rejected'
+        },
+        {
+          username: 'Greg',
+          date: new Date().toISOString(),
+          displayName: 'Greg',
+          status: 'approuved'
+        }
+      ]
+    }
+  },
+  computed: {
+    ...mapGetters([
+      'currentGroupState',
+      'ourUserIdentityContract'
+    ]),
+    hasMemberRequest () {
+      return Object.keys(this.currentGroupState.request).length > 0
```

Comment:
This currently throws an error because `this.currentGroupState.request` doesn't exist.

You can replace this with `this.requests` for now.